src/hooks/use-suggestions.ts

```diff
@@ -741,7 +741,7 @@ const getValueSuggestions = (
             default:
                 if (typeof type === 'object') {
                     // value aliases: key is the alias, value is the actual value
-                    for (const key in type) suggestion.push(key)
+                    for (const key in type) suggestion.push(`'${key}'`)
```

Comment:
If the user adds the quotes by himself, does the `key` contain the quotes ?
If so, remove any quotes from the start and from the end.

Reply:
@guyDataloop these `key`-s are supposed to come from our code, not from user input.